discovery-provider/alembic/versions/6b5186e7d28f_replace_aggregate_plays_mat_view_with_.py

```diff
@@ -0,0 +1,541 @@
+                LEFT OUTER JOIN (
+                    SELECT
+                        save_and_reposts.item_id as track_id,
+                        sum(au.follower_count) as karma
+                    FROM
+                        (
+                            select
+                                r_and_s.user_id,
+                                r_and_s.item_id
+                            from
+                                (select
+                                    user_id,
+                                    repost_item_id as item_id
+                                from
+                                    reposts
+                                where
+                                    is_delete is false AND
+                                    is_current is true AND
+                                    repost_type = 'track'
+                                union all
+                                select
+                                    user_id,
+                                    save_item_id as item_id
+                                from
+                                    saves
+                                where
+                                    is_delete is false AND
+                                    is_current is true AND
+                                    save_type = 'track'
+                                ) r_and_s
+                            join
+                                users
+                            on r_and_s.user_id = users.user_id
+                            where
+                                users.cover_photo is not null AND
+                                users.profile_picture is not null AND
+                                users.bio is not null
+                        ) save_and_reposts
+                    JOIN
+                        aggregate_user au
+                    ON
+                        save_and_reposts.user_id = au.user_id
+                    GROUP BY save_and_reposts.item_id
+                ) karma ON karma.track_id = t.track_id
+                WHERE
+                    t.is_current is True AND
+                    t.is_delete is False AND
+                    t.is_unlisted is False AND
+                    t.stem_of is Null;
+
+                CREATE INDEX trending_params_track_id_idx ON trending_params (track_id);
+            END IF;
+        END
+        $do$
+        """
+
+    conn.execute(query)
+
+
+def downgrade():
+    conn = op.get_bind()
+    query = """
+        begin;
+
+        -- drop dependent mat views and aggregate_plays table
+
+        DROP MATERIALIZED VIEW IF EXISTS trending_params;
+        DROP TABLE IF EXISTS aggregate_plays;
+
+        --- ======================= AGGREGATE USER ======================= from version 5bcbe23f6c70
+        DROP MATERIALIZED VIEW IF EXISTS aggregate_plays;
+        DROP INDEX IF EXISTS play_item_id_idx;
+
+        CREATE MATERIALIZED VIEW aggregate_plays as
```

Comment:
this seems wrong to me, is this aggregate_plays or aggregate_user? couldn't this whole chunk just be a create materialized view aggregate_plays? (like the original migration that added it)

Reply:
oops yeah i'll fix this